data_structures/reverse_queue.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <queue>
+#include <stack>
+
+using namespace std;
+
+// Utility function to print the queue
+void Print(queue<int>& Queue) {
+    while (!Queue.empty()) {
+        cout << Queue.front() << " ";
+        Queue.pop();
+    }
+}
+
+// Function to reverse the queue
+void reverseQueue(queue<int>& Queue) {
+    stack<int> Stack;
+    while (!Queue.empty()) {
+        Stack.push(Queue.front());
+        Queue.pop();
+    }
+    while (!Stack.empty()) {
+        Queue.push(Stack.top());
+        Stack.pop();
+    }
+}
+
+static void test() {}
+
```

Comment:
static void test()
{
queue<int> Queue;
for(std::size_t i=1; i<10;++i)
      Queue.push(10*i);

std::cout<<Queue<<"\n";
reverseQueue(Queue);
std::cout<<Queue<<"\n";
}

int main() {
    test();
    return 0;
}